src/templates/video_player.html

```diff
@@ -0,0 +1,43 @@
+{% extends "layout.html" %}
+
+{% block body %}
+<section class="content">
+  <div class="container">
+    <a href="/">
+      <img class="logo" src="/static/images/logo.svg" alt="Emmental Logo">
+    </a>
+    <div class="user-buttons">
+      <a href="/">
+        <button class="logout-button" type="button">Manage</button>
+      </a>
+      <a href="/logout">
+        <button class="logout-button" type="button">Logout</button>
+      </a>
+    </div>
+    <!-- needs a button for main page-->
+
+    {% with messages = get_flashed_messages() %}
+    {% if messages %}
+    <div class="flash-messages">
+      {% for message in messages %}
+      <p>{{ message|safe }}</p>
+      {% endfor %}
+    </div>
+    {% endif %}
+    {% endwith %}
+
+    <h1>Video Player:</h1>    
+    {% if video %}
+    <div class="videos">
+        <video width="75%" controls>
+          <source src="/video/{{ video }}" type="video/mp4">
+          <source src="/video/{{ video.video_id }}.{{ video.extension }}" type="video/ogg">
+          Your browser does not support HTML5 Video.
+        </video>
+    </div>
+    {% else %}
+    <h2>Video not found.</h2>
+    {% endif %}
+  </div>
+</section>
+{% endblock %}
```

Comment:
Newline at end of files is standard in a bunch of languages which is why the build earlier failed. Can you add one here?

Reply:
yup